src/tests/utests/iotc_utest_bsp_crypto_sha256.cc

```diff
@@ -0,0 +1,108 @@
+/* Copyright 2018 Google LLC
+ *
+ * This is part of the Google Cloud IoT Edge Embedded C Client,
+ * it is licensed under the BSD 3-Clause license; you may not use this file
+ * except in compliance with the License.
+ *
+ * You may obtain a copy of the License at:
+ *  https://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "gmock.h"
+#include "gtest.h"
+#include <iostream>
+using namespace std;
```

Comment:
Don't do this, especially not in the global namespace. In general, try to avoid using directives,
https://abseil.io/tips/153